core/src/models/App.ts

```diff
@@ -410,4 +408,13 @@ export class App extends CommonModel<App> {
     const redis = api.redis.clients.client;
     return redis.del(key);
   }
+
+  @AfterSave
+  @AfterDestroy
+  static async invalidateCache() {
+    AppsCache.invalidate();
+    await CLS.afterCommit(
+      async () => await redis.doCluster("api.rpc.app.invalidateCache")
```

Comment:
```suggestion
      () => redis.doCluster("api.rpc.app.invalidateCache")
```
This might work the same?